docs/user-guide/query/gremlin/gremlin-limits.md

```diff
@@ -0,0 +1,34 @@
+# GafferPop Limitations
+
+There are several restrictions with the current implementation. Many of these
+can be attributed to the fundamental differences between the two technologies
+but some features may also be yet to be implemented.
+
+Current TinkerPop features not present in the GafferPop implementation:
+
+- Property index for allowing unseeded queries.
+- Using a Gaffer user for authentication.
```

Comment:
I think this could also be generalised to also say that access control and authorisations are not supported as these are based on users.